bootrom/Makefile

```diff
@@ -3,20 +3,12 @@
 
 all: build
 
-PY := python3
-ROOT_DIR := $(shell git rev-parse --show-toplevel)
```

Comment:
The purpose was to exactly tell where the output was going, even if the `bootrom` directory moved.

Reply:
That’s fair. I had some ideas about compatibility when I changed it but really there was no reason for me to remove them so it’s good to put it back.